### PR TITLE
More test on MessagesView, and harmonize preview on Dialogs

### PR DESCRIPTION
**Pull request by bmarty**
## Type of change

- [ ] Feature
- [ ] Bugfix
- [ ] Technical
- [x] Other : ui interaction test

## Content

First commit is improving the preview on all the dialogs. The dialogs will now be displayed correctly and the same way in Showkase browser. Also add regular preview, to improve code coverage.

Next commits are adding UI interaction test on MessagesView.
Fix a bug where Event was triggered twice.

## Motivation and context

Improve code coverage and complete test on MessagesView.

image

## Tests

- That Unit test.

## Tested devices

- [ ] Physical
- [ ] Emulator
- OS version(s):

## Checklist

- [ ] Changes have been tested on an Android device or Android emulator with API 23
- [ ] UI change has been tested on both light and dark themes
- [ ] Accessibility has been taken into account. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#accessibility
- [ ] Pull request is based on the develop branch
- [ ] Pull request includes a new file under ./changelog.d. See https://github.com/element-hq/element-x-android/blob/develop/CONTRIBUTING.md#changelog
- [ ] Pull request includes screenshots or videos if containing UI changes
- [ ] Pull request includes a [sign off](https://matrix-org.github.io/synapse/latest/development/contributing_guide.html#sign-off)
- [ ] You've made a self review of your PR

### CODE REVIEW

File: features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt [hunk 1]
```diff
@@ -250,7 +250,6 @@ fun MessagesView(
         state = state.customReactionState,
         onEmojiSelected = { eventId, emoji ->
             state.eventSink(MessagesEvents.ToggleReaction(emoji.unicode, eventId))
-            state.customReactionState.eventSink(CustomReactionEvents.DismissCustomReactionSheet)
```
Comment (bmarty) [inline features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/MessagesView.kt:253]: Not needed, already done here: https://github.com/element-hq/element-x-android/blob/10060c4476695f3c0835a224ef5c7c088d143f17/features/messages/impl/src/main/kotlin/io/element/android/features/messages/impl/timeline/components/customreaction/CustomReactionBottomSheet.kt#L48

File: tests/uitests/src/test/snapshots/images/ui_S_t[l.designsystem.components.dialogs_MultipleSelectionDialogContent_null_Dialogs_MultipleSelectionDialogContent_0_null,NEXUS_5,1.0,en].png
Comment (jmartinesp) [inline tests/uitests/src/test/snapshots/images/ui_S_t[l.designsystem.components.dialogs_MultipleSelectionDialogContent_null_Dialogs_MultipleSelectionDialogContent_0_null,NEXUS_5,1.0,en].png:1]: This one looks quite broken. I think we might want to keep this one as 2 separate screenshots.

**Review by jmartinesp — APPROVED** [review]
LGTM, thanks. Just to give some more context about the dialog stuff:

The previews for dialogs were split between the actual dialogs (when they were present) and their contents because on Showkase, when using a `Dialog` component, the UI would be displayed in a new impossible to dismiss window and the user would get stuck in this screen when scrolling. I think there's no workaround for this yet, so although I don't like having redundant screenshots it's probably necessary.

File: tests/uitests/src/test/snapshots/images/ui_S_t[l.designsystem.components.dialogs_MultipleSelectionDialogContent_null_Dialogs_MultipleSelectionDialogContent_0_null,NEXUS_5,1.0,en].png
Comment (bmarty) [reply tests/uitests/src/test/snapshots/images/ui_S_t[l.designsystem.components.dialogs_MultipleSelectionDialogContent_null_Dialogs_MultipleSelectionDialogContent_0_null,NEXUS_5,1.0,en].png:1]: Yes, I have seen that too. I think this is quite acceptable for the usage we are doing with this preview (display them in Showkase).
It also shows how the dialog will render if the content height is big, so it's sort of useful :)

**Pull request merged**